src/Integration/Binding/DotNetBindingConfigProvider.cs

```diff
@@ -132,7 +139,7 @@ private async Task<IBindingConfigFile> DoGetConfigurationAsync(SonarQubeQualityP
             return serverProperties.ToDictionary(x => x.Key, x => x.Value);
         }
 
-        private RuleSet CreateRuleset(SonarQubeQualityProfile qualityProfile, Language language, IEnumerable<SonarQubeRule> rules, Dictionary<string, string> sonarProperties)
+        private CoreRuleset CreateRuleset(SonarQubeQualityProfile qualityProfile, Language language, IEnumerable<SonarQubeRule> rules, Dictionary<string, string> sonarProperties)
```

Comment:
Miscellaneous - changed to use alias for consistency (and to remove an analyzer warning)